### ISSUE TITLE
Upload Server needs to work more predictably with Editor

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/storylines-editor#55]
As of this writing, the architecture of the Editor while running the Express server is difficult to work with. When creating/editing a Storylines product, the user should be able to see immediate results. The location of the server should not matter: It could be co-located with the Editor, or on an entirely different server. 

The Upload Server should also be bundled with the Editor. The Editor can't do anything without it. 

Related #54

**Comment by dan-bowerman** [comment]
Section917 QC server is currently listening on port 6040

Contents get dumped in the following format:
https://section917.canadacentral.cloudapp.azure.com/respect/public/index-ca-en.html#/en/test

**Comment by RyanCoulsonCA** [comment]
After #64 is merged this should be done. I've merged the server code with the repo, but it needs to be run separately (instructions are in the PR).

I've implemented the loading and uploading in a way such that the server location **shouldn't** matter, but I'm not exactly sure if we'll run into CORS issues. I've added a [middleware](https://github.com/ramp4-pcar4/storylines-editor/blob/99568198ff3b76713431137df40342a5179c9818/server/index.js#L22) ([documentation](https://expressjs.com/en/resources/middleware/cors.html)) to the Express server to enable CORS, so hopefully that won't be a problem.
